qa/rpc-tests/test_framework/test_framework.py

```diff
@@ -159,6 +159,8 @@ def main(self,argsOverride=None,bitcoinConfDict=None,wallets=None):
         random.seed(self.randomseed)
         print("Random seed: %s" % self.randomseed)
 
+        self.options.tmpdir += '/' + str(self.options.port_seed)
```

Comment:
now that I look at the qa python tests code simple string concatenation is used all over the place, wonder if the test suites has never run on windows...